lib/src/widget/nav_bar/data/config_pair.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:equatable/equatable.dart';
+import 'package:xayn_design/src/widget/nav_bar/utils/config_mixin.dart';
+import 'package:xayn_design/src/widget/nav_bar/widget/nav_bar.dart';
+
+class ConfigPair extends Equatable {
+  final ConfigUpdater updater;
+  final NavBarConfigMixin? mixin;
```

Comment:
💅  config

Reply:
it is not a config. it is reference for a mixin, that can get a `config`